controllers/helmreleaseproxy/helmreleaseproxy_controller.go

```diff
@@ -224,21 +224,39 @@ func (r *HelmReleaseProxyReconciler) Reconcile(ctx context.Context, req ctrl.Req
 		return ctrl.Result{}, wrappedErr
 	}
 
-	defer func() {
-		if err := os.Remove(credentialsPath); err != nil {
-			log.Error(err, "failed to remove credentials file in path", "credentialsPath", credentialsPath)
-		}
-	}()
+	if credentialsPath != "" {
+		defer func() {
+			if err := os.Remove(credentialsPath); err != nil {
+				log.Error(err, "failed to remove credentials file in path", "credentialsPath", credentialsPath)
+			}
+		}()
+	}
+
+	caFilePath, err := r.getCAFile(ctx, helmReleaseProxy)
+	if err != nil {
+		wrappedErr := errors.Wrapf(err, "failed to get CA file for cluster")
```

Comment:
```suggestion
		wrappedErr := errors.Wrapf(err, "failed to get CA certificate file for cluster")
```
Just so we're using the same terminology "CA certificate" in both these error message.

Reply:
Thanks!